### PR TITLE
Support Kubernetes 1.24

### PR DESCRIPTION
**Pull request by kmdkuk**
Support Kubernetes 1.24. And update dependencies.
- Update go modules
- Build with go 1.19
- Update dependencies
  - Stop using nilerr

Signed-off-by: kouki <kouworld0123@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,80 +3,88 @@ module github.com/cybozu-go/cattage
 go 1.17
```
Comment (zeroalphat) [inline go.mod:3]: I think this line needs to be updated to 1.18.

Comment (kmdkuk) [reply go.mod:3]: It is not updated intentionally because we are not using the language features in the new version.

**Review by zeroalphat — DISMISSED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ Cattage depends on the specification of Argo CD and Accurate.
 Cattage supports the following versions:
 
 | Cattage | Argo CD | Accurate |
-|---------|---------|----------|
-| 0.1.x   | 2.2.5   | 0.3.x    | 
+| ------- | ------- | -------- |
+| 0.1.x   | 2.3.9   | 0.3.x    |
```
Comment (zoetrope) [inline README.md:27]: Please fix as follows:

| Cattage | Argo CD | Accurate |
| ------- | ------- | -------- |
| 0.1.3   | 2.3.9   | 0.4.x    |
| 0.1.2   | 2.3.9   | 0.3.x    |
| 0.1.0, 0.1.1   | 2.2.5   | 0.3.x    |

File: aqua.yaml [hunk 3]
```diff
@@ -8,14 +8,14 @@ registries:
     name: local
     path: registry.yaml
```
Comment (zoetrope) [inline aqua.yaml:9]: Can you update the version of aqua-registry?
Then, you can remove `registry.yaml` and 3 lines above.

**Review by zoetrope — APPROVED** [review]
LGTM

**Comment by kmdkuk** [comment]
https://github.com/cybozu-go/cattage/pull/19/commits/f792499e4e13d1ffab08c507e67c02903e7df400
I mistakenly set the following library to use v0.25.0, so I changed it back to use v0.24.7.
- k8s.io/api
- k8s.io/apimachinery
- k8s.io/client-go

@zoetrope @zeroalphat 
I'm sorry. Please review again.

**Review by zeroalphat — APPROVED** [review]
LGTM

**Review by zoetrope — APPROVED** [review]

**Pull request merged**